site_root/input/resources/examples/Patient-pt-001.json

```diff
@@ -1,15 +1,54 @@
 {
```

Comment:
@liberaliscomputing I like how you added the `identifier` in the Biospecimen resource. Can you add that here too?